torch_em/transform/generic.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, *transforms):
     def __call__(self, *inputs):
         outputs = self.transforms[0](*inputs)
         for trafo in self.transforms[1:]:
-            outputs = trafo(*outputs)
```

Comment:
This was the previous state where the outputs from the first trafo were passed as inputs to the next round of trafos using `*outputs`, however when I used this in my workflow, I noticed that `*outputs` splits the outputs (say an array of shape (3, 512, 512)) to a tuple of three different arrays (of shape (512, 512) each). I removed the `*` and it stopped this behaviour. Could you confirm if this is the way-to-go or am I missing something in the transform functions?

For reference: this happens in "raw_transform", where the first transform is the resize inputs transform, and the next is for the default raw transforms.

Reply:
`is_multi_tensor` as another flag to take care of this.